### ISSUE TITLE
[TextFields] MDCMultilineTextField clearButton is missing accessibilityLabel

### ISSUE THREAD

**Issue by material-mirror** [material-components/material-components-ios#5616]
This was filed as an internal issue. If you are a Googler, please visit [b/119140147](http://b/119140147) for more details.

---
#### Internal data
- Associated internal bug: [b/119140147](http://b/119140147)

**Comment by material-mirror** [comment]
The internal issue [b/119140147](http://b/119140147) is now closed. This issue is being closed as a result.

**Comment by felixaemiliano** [comment]
This is blocking the launch of one of our features. If you are a Googler, see b/146196400.
